internal/gamepad/gamepad.go

```diff
@@ -332,8 +411,22 @@ func (g *Gamepad) IsStandardButtonPressed(button gamepaddb.StandardButton) bool
 	if gamepaddb.HasStandardLayoutMapping(g.sdlID) {
 		return gamepaddb.IsButtonPressed(g.sdlID, button, g)
 	}
-	if g.native.hasOwnStandardLayoutMapping() {
-		return g.native.isButtonPressed(int(button))
+	m := g.native.standardButtonInOwnMapping(button)
+	switch m.kind {
+	case axisMapping:
+		return g.native.axisValue(m.index) > 35/255.0
```

Comment:
What is `35/255.0`?

Reply:
Named as a constant (and also made it 30/255, I somehow messed up copying the number from gamepaddb). Not sure where it comes from, it might just be arbitrary.